index.html

```diff
@@ -31,8 +31,53 @@
             </li>
           </ul>
         </nav>
+      </div> -->
+      <!--Start of mobile navbar-->
+      <div class="mobile-container">
+        <div>
+          <h2 class="logo">Mercy</h2>
+        </div>
+
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+        
+        <nav class="menu-mobile">
+          <ul class="nav-links-mobile">
+            <li>
+              <a href="#portfolio" class="a-links">Portfolio</a>
+            </li>
+            <li>
+              <a href="#about__me" class="a-links">About</a>
+            </li>
+            <li>
+              <a href="#contact"class="a-links">Contact</a>
+            </li>
```

Comment:
- [ ] It would be better if when a user clicks on any of the menu options, a correct part of the page is displayed. Current;y, that is not the situation. Here is a screencast of the demonstration nelow
[Screencast from 03-06-2023 11:41:56 PM.webm](https://user-images.githubusercontent.com/46388113/223281586-66244b22-a8a7-4637-ae1d-90b101f02e21.webm)